presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveArrayFunctionQueries.java

```diff
@@ -37,4 +37,13 @@ public void testRepeat()
         this.assertQueryFails("SELECT repeat(orderkey, 10001) FROM lineitem",
                 ".*Count argument of repeat function must be less than or equal to 10000.*");
     }
+
+    @Test
+    public void testShuffle()
+    {
+        this.assertQuerySucceeds("SELECT shuffle(array[NULL, 'a', 'b', 'c']), shuffle(array[-1.23, 0.0, 456.78])");
```

Comment:
This query runs on the coordinator and therefore doesn't test any logic in the worker. Let's remove it.